tests/e2e/core-tests/specs/shopper/front-end-variable-product-updates.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+/**
+ * Internal dependencies
+ */
+const {
+	shopper,
+	merchant,
+	createVariableProduct,
+} = require( '@woocommerce/e2e-utils' );
+
+let variablePostIdValue;
+
+const cartDialogMessage = 'Please select some product options before adding this product to your cart.';
+
+const runVariableProductUpdateTest = () => {
+	describe('Update variable product',() => {
+		beforeAll(async () => {
+			await merchant.login();
+			variablePostIdValue = await createVariableProduct();
+			await merchant.logout();
+		});
+
+		it('can change variable attributes to the same value', async () => {
+			await shopper.goToProduct(variablePostIdValue);
+			await expect(page).toSelect('#attr-1', 'val1');
+			await expect(page).toSelect('#attr-2', 'val1');
+			await expect(page).toSelect('#attr-3', 'val1');
+
+			await expect(page).toMatchElement('.woocommerce-variation-price', { text: '9.99' });
```

Comment:
The assertions are expecting the values set in createVariableProduct(). If these change at any stage the test will fail, perhaps we need to update both this test and createVariableProduct() to use data from the same config. This could be a separate enhancement and not a blocker for this PR.

Reply:
I agree with that! I added a new issue here for this: https://github.com/woocommerce/woocommerce/issues/29195

Reply:
Thanks!